extension/oidcextension/config.go

```diff
@@ -0,0 +1,45 @@
+// Copyright -c Google LLC
```

Comment:
huh?

Reply:
Weird I ran the license tool and got this